bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -16,21 +16,31 @@
     (url('foundation.index'), 'index', 'The Mozilla Foundation'),
     (url('foundation.about'), 'about', 'About the Foundation'),
     ('https://mzl.la/foundation-strategy', 'strategy', '2016-2018 Strategy'),
-    (url('foundation.annualreport'), 'report', 'Annual Report'),
-    ('https://careers.mozilla.org/', 'careers', 'Careers'),
-    (donate_url('foundation_nav'), 'donate', 'Donate'),
     (url('foundation.licensing'), 'licensing', 'Licensing & Trademarks'),
     (url('foundation.documents.index'), 'documents', 'Public Documents'),
+    ('https://careers.mozilla.org/', 'careers', 'Careers'),
 ] %}
 
-{% block content %}
-  <main id="main-content">
-    {% block article_title %}{% endblock %}
-
-    <div class="main-column">
-      {% block article %}{% endblock %}
-    </div>
+{% set programs = [
+  (url('foundation.issues'), 'issues', 'Key Internet Issues'),
+  (url('foundation.advocacy'), 'advocacy', 'Advocacy'),
+  (url('foundation.leadership-network'), 'leadership-network', 'Leadership Network')
+] %}
 
+{% block content %}
+  <main>
+    <header class="page-header">
+      <h1><span class="content">{{ _('The Mozilla Foundation') }}</span></h1>
+    </header>
+    <section>
+      <div class="content content-main">
+      <div class="main-column">
+        <header class="section-header">
+        <h1>{% block article_title %}{% endblock %}</h1>
```

Comment:
Nitpick: With the `h1` above this should semantically be an `h2` and all subsequent headings ranked down accordingly. The proposed HTML5 outlining algorithm a few years ago led us to believe it was kosher to begin a new heading rank at each new sectioning root, but that algorithm is now officially abandoned so we probably need to stick to classic heading rank.

Also, I see you're using `:empty` to hide these elements if they have no content, but that's pretty much presentational; the empty element still exists in the markup, which isn't ideal. I'd suggest a single block like `{% block article_head %}` and putting whatever markup you need into that block in the page templates.